advocacy_docs/edb-postgres-ai/analytics/how_to_lakehouse_sync.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Lakehouse Sync
+navTitle: Lakehouse Sync
+description: How to perform a Lakehouse Sync.
+deepToC: true
+---
+
+## Overview
+
+Performing a Lakehouse Sync is a way to capture information from a transactional database at a point in time and sync that information to a Managed Store Location (MSL).
+
+The Lakehouse sync process organizes the transactional database data into Lakehouse tables stored in the MSL. This process allows the data to be queried by a Lakehouse node, which is optimized for higher-performance queries using a vectorized query engine designed for Lakehouse tables.
+
+## Performing a Lakehouse Sync
+
+### Prerequisites
+
+- a Postgres cluster hosted and managed by EDB Postgres AI® Cloud Service
+
+### Navigate to Lakehouse Sync
+
+1. Go to the [EDB Postgres AI Console]().
+
+2. From the landing page, select the project with the database instance you want to sync. If it is not shown on the landing page, select the **View Projects** link in the **Projects** section and select your project from there.
+
+3. Select the **Migrate** dropdown in the left navigation bar and then select **Migrations**.
+
+4. Select the **Create New Migration** button.
+
+### Define Lakehouse Sync
+
+5. Give the sync a **Name**, then select a **Source Cluster** and the **Database** you want to sync.
+
+6. If you have already created an MSL you want to use, select that MSL from the list of available MSLs and move on to [Selecting Tables](#selecting-tables) below. If not, select the **Create New Managed Storage Location** button to open the **Add Managed Storage Location** dialog.
+
+7. Select the AWS region for the new MSL.
+
+8. Set a location prefix in the form near the bottom of the **Add Managed Storage Location** dialog to complete the definition of the MSL. A location prefix is a unique name used to identify any resources and assets associated with the MSL.
+
+![List of MSLs](./images/msl_list.png)
+
+9. Select the **Create Managed Storage Location** button.
+
+### Selecting Tables
+
+10. Select the **Tables** tab next to the **Get Started** tab near the top of the page and select which tables and columns you wish to be included in the migration.
+
+### Start Lakehouse Sync
+
+11. Select the **Start Lakehouse Sync** button.
+
+12. If successful, you will see your Lakehouse sync with the 'Creating' status under 'MOST RECENT' migrations on the Migrations page. The time taken to perform a sync can depend upon how much data is being synchronized and may take several hours. 
```

Comment:
The first sync in a project will take significantly longer (at least 1 hour) due to provisioning Kafka infrastructure. This also applies when you are performing a sync after a long time without one, because we shut down idle Kafka infrastructure (in fact, it might be after 24 hours of non-use.... not sure.)

Reply:
Okay I added an admonition regarding the infrastructure provisioning time the first sync of a project.